CMakeLists.txt

```diff
@@ -1845,14 +1847,46 @@ TARGETS
     libwebrtcbuild
     libyuv
     ${platform_export}
-NAMESPACE
-    tg_owt::
-FILE
-    "${CMAKE_CURRENT_BINARY_DIR}/tg_owtTargets.cmake"
+)
+
+export(
+    TARGETS ${export_targets}
+    NAMESPACE tg_owt::
+    FILE "${CMAKE_CURRENT_BINARY_DIR}/tg_owtTargets.cmake"
 )
 
 configure_file(
     "cmake/tg_owtConfig.cmake"
     "${CMAKE_CURRENT_BINARY_DIR}/tg_owtConfig.cmake"
     COPYONLY
 )
+
+target_include_directories(tg_owt
+PUBLIC
+    $<BUILD_INTERFACE:${webrtc_loc}>
+    $<INSTALL_INTERFACE:${webrtc_includedir}>
+)
+
+install(
+    TARGETS ${export_targets}
+    EXPORT tg_owtTargets
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
```

Comment:
If there would be all internal static dependencies, maybe install them into some private directory, e.g. `${CMAKE_INSTALL_LIBDIR}/tg_owt`?

Reply:
No. It only produces one library named `libtg_owt.a` containing all objects.

Reply:
Ok